### PR TITLE
Add staff team

### PR DESCRIPTION
**Pull request by Spit-fires**
I added staff team page

### CODE REVIEW

File: staff.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                <p>Board of Director</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+              <h2 class="text-lg">corn pops#0949</h2>
+              <p>Board of Director</p>
+            </div>
+            <div class="bg-primary p-1 rounded">
+              <h2 class="text-lg">Luke M#6078</h2>
+              <p>Board of Director</p>
+            </div>
```
Comment (JaronZ) [inline staff.html]: Luke is added twice
```suggestion
```

File: staff.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                <li>
+                    <a href="commands">Commands</a>
+                </li>
+                <li>
+                    <a href="https://beta.statcord.com/bot/709705136259334296" target="_blank" rel="noopener noreferrer">Statistics</a>
+                </li>
+            </ul>
+        </div>
+        <div class="flex justify-center gap-1">
+            <img src="https://github.com/PixelPizza/Images/blob/main/b3ae66a1d6f601eb55114fbd172ad973.png?raw=true">
```
Comment (JaronZ) [inline staff.html:110]: Could you make this into a local image? so it doesn't have to load it from github

**Review by JaronZ — CHANGES_REQUESTED** [review]

File: staff.html (hunk 2 above)
Comment (JaronZ) [reply staff.html:110]: Nvm, I'll do it myself for all of them

Comment (Spit-fires) [reply staff.html:110]: Ok you can do it if you want

**Review by JaronZ — APPROVED** [review]

**Pull request merged**